src/common/options/mds.yaml.in

```diff
@@ -119,11 +119,12 @@ options:
 - name: mds_cache_reservation
   type: float
   level: advanced
-  desc: amount of memory to reserve for future cached objects
-  fmt_desc: The cache reservation (memory or inodes) for the MDS cache to maintain.
+  desc: Amount of memory to reserve for future cached objects
+  fmt_desc: The percentage of the configured cache size to reserve.
     Once the MDS begins dipping into its reservation, it will recall
     client state until its cache size shrinks to restore the
-    reservation.
+    reservation. In other words, (1 - this value ) is a cache fullness
+    high water mark above which client caps will be recalled.
```

Comment:
The existing text seemed a bit unclear to me, I've adjusted to what I think it's trying to say, please validate.